packages/function-tree/src/index.js

```diff
@@ -191,6 +191,11 @@ export class FunctionTree extends EventEmitter {
     this.runTree.on = this.on.bind(this)
     this.runTree.once = this.once.bind(this)
     this.runTree.off = this.removeListener.bind(this)
+
+    // When not extended
+    if (!returnInstance) {
```

Comment:
Ah, I see what you mean... problem is Node, in Node we want this API:

```js
const FunctionTree = require('function-tree').FunctionTree
```

Doing:
```js
const FunctionTree = require('function-tree').default
```

Is just super weird to me... hm... what do others do with this? I have never seen a library where you have to require `.default`